frontend/src/pages/privacy.vue

```diff
@@ -58,13 +76,27 @@ export default defineComponent({
   layout: "content-layout",
   setup() {
     const i18n = useI18n()
+    const { $cookies } = useContext()
+    const featureFlagStore = useFeatureFlagStore()
 
     useMeta({
       title: `${i18n.t("privacy.title", {
         openverse: "Openverse",
       })} | Openverse`,
       meta: [{ hid: "robots", name: "robots", content: "all" }],
     })
+
+    const handleChange = ({ checked }: { checked: boolean }) => {
+      featureFlagStore.toggleFeature("analytics", checked ? ON : OFF)
+      $cookies.set("features", featureFlagStore.flagStateMap)
+    }
+
+    return {
+      ON,
+      handleChange,
+      getFlagStatus,
+      featureState: featureFlagStore.featureState,
+    }
```

Comment:
Nit: Maybe instead of passing `ON` and `featureState` it would be more direct to pass something like an `isEnabled` composable with a value of `featureState('analytics') === ON`?